edx_sailthru/sailthru_content/services/sailthru_translation_service.py

```diff
@@ -87,10 +87,10 @@ def _create_course_vars(self, course, course_run, url, site_name, program_dictio
             sailthru_content_vars['course_type'] = 'gfa'
 
         # If the course runs have associated programs, also append programs information
-        course_programs = course.get('programs')
-        if course_programs and program_dictionary:
+        course_run_programs = course_run.get('programs')
```

Comment:
I don't think. Because API should be tested in course-discovery, which I believe already tested.